runtime/shared_common/OSCache.hpp

```diff
@@ -109,6 +109,7 @@ typedef struct SH_OSCache_Info {
         UDATA isCompatible; /** Is the cache compatible with this VM */
         UDATA isCorrupt; /** Is set when the cache is found to be corrupt */
         UDATA isJavaCorePopulated; /** Is set when the javacoreData contains valid data */
+         I_8 layer; /** cache layer number */
```

Comment:
This looks mis-aligned with the other declarations.

Reply:
Fixed